src/SignalR/server/Core/src/Internal/HubMethodDescriptor.cs

```diff
@@ -110,7 +110,7 @@ public HubMethodDescriptor(ObjectMethodExecutor methodExecutor, IServiceProvider
                         }
                         else
                         {
-                            throw new InvalidOperationException($"This service provider doesn't support keyed services.");
+                            throw new InvalidOperationException($"Unable to resolve {nameof(FromKeyedServicesAttribute)}. This service provider doesn't support keyed services.");
```

Comment:
This message isn't completely accurate. We **are** resolving `FromKeyedServicesAttribute`. We just can't get the service it's referencing.

Maybe something like `"Unable to resolve (keyed?) service referenced by FromKeyedServicesAttribute. This service provider doesn't support keyed services."`

Reply:
It's a small improvement. I'll update it in SignalR and minimal API.